src/mon/OSDMonitor.cc

```diff
@@ -8312,6 +8312,21 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     wait_for_finished_proposal(op, new Monitor::C_Command(mon, op, 0, rs,
 					      get_last_committed() + 1));
     return true;
+  } else if (prefix == "osd get-full-ratio" ||
```

Comment:
All these commands are read-only. Please move them to `preprocess_command()`.

Reply:
@jecluis done !